docs/querying/sql-scalar.md

```diff
@@ -223,8 +223,8 @@ The [DataSketches extension](../development/extensions-core/datasketches-extensi
 
 |Function|Notes|
 |--------|-----|
-|`HLL_SKETCH_ESTIMATE(expr[, round])`|Returns the distinct count estimate from an HLL sketch. `expr` must return an HLL sketch. The optional `round` boolean parameter will round the estimate if set to `true`, with a default of `false`.|
-|`HLL_SKETCH_ESTIMATE_WITH_ERROR_BOUNDS(expr[, numStdDev])`|Returns the distinct count estimate and error bounds from an HLL sketch. `expr` must return an HLL sketch. An optional `numStdDev` argument can be provided.|
+|`HLL_SKETCH_ESTIMATE(expr[, round])`|Returns the distinct count estimate from an HLL sketch. `expr` must return an HLL sketch. The optional `round` boolean argument rounds the estimate if set to `true`, with a default of `false`.|
```

Comment:
```suggestion
|`HLL_SKETCH_ESTIMATE(expr[, round])`|Returns a distinct count estimate from an HLL sketch. `expr` must be an HLL sketch. To round the estimate, set `round` to true. Otherwise, `round` defaults to false.|
```